apps/music/elements/music-scan-progress.js

```diff
@@ -115,17 +115,28 @@ proto.detachedCallback = function() {
 };
 
 proto.attributeChangedCallback = function(attr, oldVal, newVal) {
-  switch (attr) {
-    case 'value':
-      this.els.value.textContent = newVal;
-      break;
-    case 'heading':
-      this.els.heading.textContent = newVal;
-      break;
-    case 'subheading':
-      this.els.subheading.textContent = newVal;
-      break;
+  var promise;
+  var l10nId = this.getAttribute(attr + '-l10n-id');
+
+  if ((newVal === null || newVal === undefined) && l10nId) {
```

Comment:
I think we also need to check for an empty string here because this is an attribute value. Just do `if (!newVal && l10nId)`.